src/flag_gems/ops/isclose.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+import torch
+import triton
+import triton.language as tl
+
+from ..utils import pointwise_dynamic
+from .all import all
+
+
+@pointwise_dynamic(
+    is_tensor=[True, True, False, False, False, False],
+    promotion_methods=[(0, 1, "ALWAYS_BOOL")],
+)
+@triton.jit
+def isclose_func(
+    x,
+    y,
+    rtol,
+    atol,
+    equal_nan: tl.constexpr,
+    zero_tol: tl.constexpr,
+):
+    cast_x = x if x.dtype.is_fp64() else x.to(tl.float32)
+    cast_y = y if x.dtype.is_fp64() else y.to(tl.float32)
+    if x.dtype.is_bf16():
```

Comment:
Should fp16 also use (cast_x == cast_y) to compute closeness?

Reply:
> Should fp16 also use (cast_x == cast_y) to compute closeness?

This will cause coredump...